src/TropicalGeometry/groebner_basis.jl

```diff
@@ -296,7 +296,7 @@ julia> desimulate_valuation(wSim, nuMax; perturbation=uSim)
 
 ```
 """
-function desimulate_valuation(w::Vector{QQFieldElem}, nu::TropicalSemiringMap{K,p,typeof(min)}; perturbation::Union{Nothing,Vector}=nothing) where {K,p}
+function desimulate_valuation(w::AbstractVector{QQFieldElem}, ::TropicalSemiringMap{K,p,typeof(min)}; perturbation::Union{Nothing,AbstractVector}=nothing) where {K,p}
     @req w[1]<0 "invalid weight vector"
     # scale the vector so that first entry is 1, then remove first entry
     w ./= w[1]
```

Comment:
This does not work for (every) `AbstractVector`, as far as I can see:
```
julia> w = 1:2
1:2

julia> w isa AbstractVector
true

julia> w ./= w[1]
ERROR: CanonicalIndexError: setindex! not defined for UnitRange{Int64}
Stacktrace:
[...]
```
(If the elements of `w` are `QQFieldElem`s, the error is weirder...)